client/client.go

```diff
@@ -18,11 +18,11 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package cluster
+package client
 
 import (
 	"github.com/m3db/m3cluster/kv"
-	"github.com/m3db/m3x/watch"
+	xwatch "github.com/m3db/m3x/watch"
```

Comment:
can we get rid of the alias here? the package name in m3x/watch is already xwatch